qa/tasks/vstart_runner.py

```diff
@@ -1003,18 +1004,23 @@ def exec_test():
             create_cluster = True
         elif f == "--create-cluster-only":
             create_cluster_only = True
+        elif f == "--skip-binaries-checking":
+            skip_binaries_checking = True
         else:
             log.error("Unknown option '{0}'".format(f))
             sys.exit(-1)
 
-    # Help developers by stopping up-front if their tree isn't built enough for all the
-    # tools that the tests might want to use (add more here if needed)
-    require_binaries = ["ceph-dencoder", "cephfs-journal-tool", "cephfs-data-scan",
-                        "cephfs-table-tool", "ceph-fuse", "rados"]
-    missing_binaries = [b for b in require_binaries if not os.path.exists(os.path.join(BIN_PREFIX, b))]
-    if missing_binaries:
-        log.error("Some ceph binaries missing, please build them: {0}".format(" ".join(missing_binaries)))
-        sys.exit(-1)
+    if skip_binaries_checking:
+        log.warn('SKIPPED checking: required ceph binaries.')
+    else:
+        # Help developers by stopping up-front if their tree isn't built enough for all the
+        # tools that the tests might want to use (add more here if needed)
+        require_binaries = ["ceph-dencoder", "cephfs-journal-tool", "cephfs-data-scan",
+                            "cephfs-table-tool", "ceph-fuse", "rados"]
+        missing_binaries = [b for b in require_binaries if not os.path.exists(os.path.join(BIN_PREFIX, b))]
+        if missing_binaries:
+            log.error("Some ceph binaries missing, please build them: {0}".format(" ".join(missing_binaries)))
+            sys.exit(-1)
```

Comment:
how about `exit(1)` only if `not skip_binaries_checking`?

Reply:
OK, should we keep teh same log.error meassage but trasnforming it into a log.warn ?

Reply:
i am okay either way. as i don't think `log.error()` is as fatal as `exit(1)`.

Reply:
OK. Renamed flag to: --ignore-missing-binaries